jdaviz/configs/specviz/plugins/line_analysis/line_analysis.py

```diff
@@ -293,10 +294,21 @@ def _uncertainty(result):
                     flux_unit = u.Unit(self.spectrum_viewer.state.y_display_unit)
                 else:
                     add_flux = False
+
+                solid_angle_in_flux_unit = check_if_unit_is_per_solid_angle(flux_unit,
+                                                                            return_unit=True)
+                if solid_angle_in_flux_unit is None:
+                    # use sr as a placeholder unit. is_equivalent() checks won't
+                    # pass anyway if theres no solid angle in the unit, so it
+                    # won't matter what this is
+                    solid_angle_in_flux_unit = u.sr
```

Comment:
if there isn't another way around this, how about `u.dimensionless_unscaled` instead just to avoid any confusion with other logic that might make use of this in the future?